Integrations/integration-fireeye.yml

```diff
@@ -110,7 +127,7 @@ script:
                 }
             ],
         },
-        'fe-submit': {
+        'fe-submit-file': {
```

Comment:
@sgosai I think it might be better to keep this as before for backwards compatibility